system/src/system_network_manager.cpp

```diff
@@ -1064,7 +1066,7 @@ int NetworkManager::powerInterface(if_t iface, bool enable) {
         if (ifState->pwrState != IF_POWER_STATE_UP && ifState->pwrState != IF_POWER_STATE_POWERING_UP) {
             ifState->pwrState = IF_POWER_STATE_POWERING_UP;
         }
-        LOG(TRACE, "Request to power on the interface");
+        LOG(TRACE, "Request to power on interface %d", index);
```

Comment:
This is just to help understand which network interface is being powered on now that its ambiguous